lib/services/registered_addons.pm

```diff
@@ -33,6 +33,7 @@ sub suseconnect_ls {
 sub check_registered_system {
     my ($system) = @_;
     my $pro = uc get_var('SLE_PRODUCT');
+    $pro = 'SLE_' . $pro if ($pro eq 'HPC');
```

Comment:
Here $pro should be 'hpc', so would you update it to $pro eq 'hpc'?

Reply:
When I get SLE_PRODUCT is hpc , I will change it to SLE_HPC, so it will be same with SUSEConnect output